src/components/seo.js

```diff
@@ -25,32 +25,40 @@ function SEO({ description, lang, meta, keywords, title }) {
       meta={[
         {
           name: `description`,
-          content: metaDescription,
+          content: 'A place for Q&A and free mentorship',
+        },
+        {
+          property: `og:title`,
+          content: 'Mentor Night | Free Code Camp',
+        },
+        {
+          property: `og:description`,
+          content: 'A place to be coached or to coach for free along your developer journey',
```

Comment:
A minor point, but I think "A place to coach or be coached for free along your developer journey" rolls off the tongue a little easier. We may also want to reuse `metaDescription` from before as well.